src/CrowdDesign/Views/Project/EditProject.cshtml

```diff
@@ -20,29 +20,31 @@
     <table id="tblMorphChart">
         <thead>
             <tr>
-                <th class="tdDimension">Dimensions</th>
+                <th>Dimensions</th>
                 @if (Model != null && Model.Dimensions != null)
                 {
                     if (Model.Dimensions.Any(c => c.Sketches != null && c.Sketches.Count > 0))
                     {
                         int proposedSolutionsCount = (from c in Model.Dimensions
-                                                      where c.Sketches != null && c.Sketches.Count > 0
-                                                      select c.Sketches.Count()).Max();
+                            where c.Sketches != null && c.Sketches.Count > 0
+                            select c.Sketches.Count()).Max();
 
                         if (proposedSolutionsCount > 0)
                         {
-                            for (int i = 1; i <= proposedSolutionsCount; i++)
-                            {
-                                string name = "Solution Alternative " + i;
-
-                                <th>@name</th>
-                            }
+                            int i = proposedSolutionsCount + 1;
+                            <th colspan="@i">Solution Alternatives</th>
```

Comment:
Small change to make the code shorter: 

declare int i = 1 in line 25;
change line 34 to i += proposedSolutionsCount;
remove line 35;
remove line 38 (else);
change line 40 for <th colspan=.... >